src/libraries/System.Security.Cryptography.Algorithms/src/Internal/Cryptography/AesImplementation.cs

```diff
@@ -43,6 +43,50 @@ protected sealed override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+        protected override bool TryDecryptEcbCore(
+            ReadOnlySpan<byte> ciphertext,
+            Span<byte> destination,
+            PaddingMode paddingMode,
+            out int bytesWritten)
+        {
+            UniversalCryptoTransform transform = CreateTransformCore(
+                CipherMode.ECB,
+                paddingMode,
+                Key,
+                iv: null,
+                blockSize: BlockSize / BitsPerByte,
+                paddingSize: BlockSize / BitsPerByte,
+                0, /*feedback size */
+                encrypting: false);
+
+            using (transform)
+            {
+                return transform.TransformOneShot(ciphertext, destination, out bytesWritten);
+            }
```

Comment:
(Resolved in my head before submitting, but writing it down for posterity)

This code surprises me, it looks like it would just work when implemented on the base class, since it's just creating the transform object and calling go on it.  Isn't that the same as just calling CreateEncryptor().TransformFinalBlock?

(Resolution: No, it's not the same, because the method forces ECB but CreateEncryptor uses the Mode property)